library/src/interactor_impl.cxx

```diff
@@ -975,6 +999,21 @@ interactor& interactor_impl::initBindings()
     }
   };
 
+  // "Cycle anti-aliasing" , "NONE/FXAA/SSAA"
+  auto docAA = [&]()
+  {
+    std::string desc;
+    if (!this->Internals->Options.render.effect.antialiasing.enable)
+    {
+      desc = "none";
+    }
+    else
+    {
+      desc = this->Internals->Options.render.effect.antialiasing.mode;
```

Comment:
you should be able to cover this by enabling it and showing cheatheet. maybe use an existing test